contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -136,52 +136,37 @@ def _execute(self, track_index: int = 0, new_index: int = 0, expected_new_index:
                        "Expected track index {} after reordering, got {}".format(expected_new_index, index))
 
 
-class MatchTracks(CaptureWindowE2ETestCaseBase):
+class VerifyTracksExist(CaptureWindowE2ETestCaseBase):
     """
     Verify that the existing visible tracks match the expected tracks
```

Comment:
This is not true anymore, isn't it? We don't care about other tracks in this test case, do we?

Reply:
Yeah, that's why I usually dislike comments... I've updated the comments of the affected classes, PTAL.